### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by retailify**

### CODE REVIEW

**Comment by rfleschenberg** [comment]
Hi Thomas. Thank you for the PR! I would like so suggest some modifications:

* For the plural of _Panel_, we should use _Panels_ or _Panel_. duden.de suggest _Panels_.
* For _Heading Size_, use _Schriftgröße (Überschrift)_ or _Größe: Überschrift_ or something like that.
* There is a typo for _Link_ (_Ling_).
* We should use a hyphen for terms like _Button-Style_, _Button-Größe_ and so on (_durchkoppeln_), or maybe just use a compound word (_Buttonstyle_). A space always separates words.
* There is a typo for _Nicht floaten_ (_Nich_).
* Wrap: "Umbrechen"?
* Typo for _Geräte_ (_Geräder_)
* Typo at _Spaten Einheiten_ (also, make it one word: _Spalteneinheiten_).
* Typo at _Kein Offste_.

Can you update the PR to fix these?

**Comment by retailify** [comment]
Yes, I'll fix the errors.

**Comment by rfleschenberg** [comment]
Ping ;)

Are you still working on this, Thomas? If not, no problem, we can just fix the errors ourselves.

File: cmsplugin_cascade/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -99,543 +99,543 @@ msgstr "In '%(label)s': Wert '%(value)s' für Feld '%(field)s' muss numerisch se
...
 
 #: bootstrap3/accordion.py:42
 msgid "Close others"
-msgstr "Schließe die anderen"
+msgstr "Schließe die Anderen"
 
 #: bootstrap3/accordion.py:44
 msgid "Open only one panel at a time."
-msgstr ""
+msgstr "Öffne nur eine Panele zur Zeit."
```
Comment (bittner) [inline cmsplugin_cascade/locale/de/LC_MESSAGES/django.po]: msgstr "Immer nur ein Panel öffnen."

**Comment by bittner** [comment]
Probably it would be easier to manage translations in Transifex. The review process there is better suited for translators, so they can focus on the content not the `.po` file code.

**Comment by rfleschenberg** [comment]
Using Transifex sounds like a good idea to me. If you want to submit a PR that adds Transifex support, please go ahead :)

**Pull request merged**